environment.yml

```diff
@@ -0,0 +1,14 @@
+name: interpret-compression
+channels:
+- conda-forge
+dependencies:
+- conda-forge::python=3.6
+- conda-forge::pandas=0.22.0
+- conda-forge::numpy=1.14.2
+- conda-forge::seaborn=0.8.1
+- conda-forge::jupyter=1.0.0
+- conda-forge::ipykernel=4.8.1
+- conda-forge::nb_conda_kernels=2.1.0
+- conda-forge::openpyxl=2.5.1
+- pip:
+  - hetio==0.2.3
```

Comment:
I'd update hetio to 0.2.8. it should have several new features and bug fixes since 0.2.3

Reply:
awesome, I will update accordingly